### PR TITLE
docs: fix typo

### PR DESCRIPTION
**Pull request by wzyboy**

### CODE REVIEW

File: src/borg/archiver/config_cmd.py [hunk 1]
```diff
@@ -146,7 +146,7 @@ def build_parser_config(self, subparsers, common_parser, mid_common_parser):
         For security reasons, this command only works on local repositories.
 
         To delete a config value entirely, use ``--delete``. To list the values
-        of the configuration file or the default values, use ``--list``.  To get and existing
```
Comment (ThomasWaldmann) [inline src/borg/archiver/config_cmd.py:149]: Thanks for the PR!

As documented in the first lines of borg-config.1 and config.rst.inc, these 2 files are generated.

Thus, config_cmd.py is the only file that needs to be changed to fix this typo. Can you remove the other files from this PR, please?

Comment (wzyboy) [reply src/borg/archiver/config_cmd.py:149]: Done!

**Comment by ThomasWaldmann** [comment]
Can you please check whether the same typo is in 1.2-maint and 1.4-maint and if so, backport the fix to these branches? Likely, you only need to change the `borg/archiver.py` file there.

**Comment by ThomasWaldmann** [comment]
Thanks!

**Pull request merged**